### ISSUE TITLE
Kabyle locale

### ISSUE THREAD

**Issue by belkacem77** [Tatoeba/tatoeba2#1610]
Please don't forget to plan to sync the kab locale from transifex. Translation is ongoing

**Comment by trang** [comment]
@belkacem77 Thanks for working on the Kabyle localisation.

I have updated our [dev website](https://dev.tatoeba.org/) to support this new language. You can select it from the dropdown in the top-right corner (you should see the option "Taqbaylit").

The dev website is updated regularly to include new translations from Transifex (every 30 minutes). So after you've added new translations, you can go check on the dev website to see if they really fit or not.

We will activate Kabyle on the real website once there is enough translations. You want to focus on the resources `languages.pot` (ideally 100% translated) and `default.pot` (90% should be enough). The other ones are less important.

Also, at the moment I'm noticing a bug for the list of languages. The names simply don't appear, it's all blank... Not sure why. We'll have to fix that as well before we can enable Kabyle on the real website.

**Comment by trang** [comment]
@belkacem77 I'm not sure if you closed by mistake, but I re-opened it and will leave it open till Kabyle is enabled on the real website.

**Comment by belkacem77** [comment]
@trang Sorry for closing  it. I tought tha's enough to close it. 
I'm on the transifex, it should be fully translated within next week.
It comes on the right time since I have a training session by next  03 or 04 about it (Tatoeba). We need to create, hots and share Kabyle corpus for some NLP coming projects in the future.

**Comment by S-Amiri** [comment]
@trang thank you so much we are going to continue the translation

**Comment by trang** [comment]
I found out the issue regarding the empty language lists.

The string `{language}` was translated into `{tutlayt}`. I changed it to `{language}`. 

This is a special string that is used for languages that have declensions. If Kabyle doesn't have declensions, simply leave it as `{language}`. Otherwise please have a look at our [wiki article](https://en.wiki.tatoeba.org/articles/show/interface-translation#declensions) on this topic.

**Comment by belkacem77** [comment]
It would be an error. We understand that anything in brackets {} is a variable. I should check again tha traslation. Thanks for correction.

**Comment by belkacem77** [comment]
Hi @trang 
The needed files are translated. For the language, as we worked on CLDR, ISO 639-2 languages are included. For the others we kept them as they are in English. Another team is working on the list of the iso 639-3 languages.

I tested the translation on the dev site. Some errors were corrected. If we got others while using the Website, we will correct them. I think we can launch Kab UI on production.
